guides/advanced_guides/settings.md

```diff
@@ -0,0 +1,280 @@
+
+## Ranking rules
+
+Built-in ranking rules that **ensure relevancy in search results**. Ranking rules are applied in a default order which can be changed in the settings. You can add or remove rules and change their order of importance.
+
+`rankingRules=[<String>, <String>, ...]`
+
+- `[<String>, <String>, ...]` (Array of strings, see default value below)
+
+  An array of strings that contains the ranking rules sorted by order of importance (arranged from the most important rule to the least important rule).
+
+Default value:
+
+```json
+["typo", "words", "proximity", "attribute", "wordsPosition", "exactness"]
+```
+
+By default, rules are executed in the following order:
+
+#### 1. Typo
+
+Results are sorted by **increasing number of typos**: find documents that match query terms with fewer typos first.
+
+#### 2. Words
+
+Results are sorted by **decreasing number of matched query terms** in each matching document: find documents that contain more occurrences of the query terms first.
+
+#### 3. Proximity
+
+Results are sorted by **increasing distance between matched query terms**: find documents that contain more query terms found close together (close proximity between two query terms) and appearing in the original order specified in the query string first.
+
+#### 4. Attribute
+
+Results are sorted according to **[the order of importance of the attributes](/guides/main_concepts/relevancy.md#importance-of-the-attributes)**: find documents that contain query terms in more important attributes first.
+
+#### 5. Words Position
+
+Results are sorted by **the position of the query words in the attributes**: find documents that contain query terms earlier in their attributes first.
+
+#### 6. Exactness
+
+Results are sorted by **the similarity of the matched words with the query words**: find documents that contain exactly the same terms as the ones queried first.
+
+[Learn more about ranking rules](/guides/main_concepts/relevancy.md)
+
+#### Example
+
+To add your ranking rules to the settings, send:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "rankingRules": [
+          "typo",
+          "words",
+          "proximity",
+          "attribute",
+          "wordsPosition",
+          "exactness",
+          "asc(release_date)",
+          "desc(rank)"
+      ]
+  }'
+```
+
+With the settings in the example above, documents will be sorted by number of typos first. If too many documents have the same number of typos, the `words` rule will be applied. This operation will be repeated with the next rules until the number of documents is limited to the requested number of documents (default: 20).
```

Comment:
gathered/achieved/obtained
```suggestion
With the settings in the example above, documents will be sorted by number of typos first. If too many documents have the same number of typos, the `words` rule will be applied. This operation will be repeated with the next rules until the requested number of documents has been obtained (default: 20).
```

Reply:
Would you prefer `reached` or `obtained`?

Reply:
Which one do you prefer?